editoast/editoast_derive/src/model/args.rs

```diff
@@ -134,4 +134,6 @@ pub(super) struct ModelFieldArgs {
     pub(super) remote: Option<syn::Type>,
     #[darling(default)]
     pub(super) uom_unit: Option<syn::Path>,
+    #[darling(default)]
+    pub(super) non_null_array: Option<syn::Type>,
```

Comment:
```suggestion
    pub(super) non_null_array: Option<syn::Path>,
```

allows qualified types such as `std::vec::Vec`